client/state/themes/utils.js

```diff
@@ -4,9 +4,85 @@
  * External dependencies
  */
 import startsWith from 'lodash/startsWith';
+import { omit, omitBy } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { DEFAULT_THEME_QUERY } from './constants';
+
+/**
+ * Constants
+ */
+const REGEXP_SERIALIZED_QUERY = /^((\d+):)?(.*)$/;
```

Comment:
The first set of brackets can be a non-capturing group here I think, if you want:
`/^(?:(\d+):)?(.*)$/`